src/strategies/index.ts

```diff
@@ -691,7 +692,8 @@ const strategies = {
   'erc3525-vesting-voucher': erc3525VestingVoucher,
   'xrook-balance-of-underlying-weighted': xrookBalanceOfUnderlyingWeighted,
   'orbs-network-delegation': orbsNetworkDelegation,
-  'erc721-pair-weights': erc721PairWeights
+  'erc721-pair-weights': erc721PairWeights,
+  'prepo-vesting': prepoVesting
```

Comment:
better to add it somewhere not at the very end, otherwise it'll very likely have a merge conflict
```suggestion
  'prepo-vesting': prepoVesting,
  'erc721-pair-weights': erc721PairWeights
```